include/circt/Dialect/SV/SVStatements.td

```diff
@@ -527,6 +527,20 @@ def CoverOp : SVOp<"cover", [ProceduralOp]> {
     "custom<OmitEmptyStringAttr>($label) attr-dict $property `:` type($property)";
 }
 
+def CoverConcurrentOp : SVOp<"cover.concurrent", []> {
+  let summary = "concurrent cover statement, i.e., cover property";
+  let description = [{
+    Assert that a specific property happens during the course of execution.
```

Comment:
Please add a cite for the section of the SV standard.

Reply:
Added.